resources/prefs.json

```diff
@@ -124,5 +124,13 @@
   "shell.native-orientation": "both",
   "shell.native-titlebar.enabled": true,
   "shell.searchpage": "https://duckduckgo.com/html/?q=%s",
+  "threadpools.async_runtime_workers.fallback": 4,
```

Comment:
All fallback could be one I think

Reply:
I guess we can  always come back to this and make it a per pool option again, if there a specific need comes up. 
For now, I guess it just simplifies things if we only have one shared option as you suggest.